lib/results_page.dart

```diff
@@ -0,0 +1,276 @@
+  ),
+  const ProfileCard(
+    id: 3,
+    companyName: "Green Landscaping",
+    ownerName: "HEHEHEH",
+    image: "/placeholder.svg?height=100&width=100",
+    rating: 4,
+    tags: ["Landscaping", "Lawn Care", "Tree Trimming"],
+    phone: '', email: '',
+    aboutUs: 'blah blah blah',
+  ),
+  const ProfileCard(
+    id: 4,
+    companyName: "Handy Home Repairs",
+    ownerName: "Mike Johnson",
+    image: "/placeholder.svg?height=100&width=100",
+    rating: 4.8,
+    tags: ["General Repairs", "Carpentry", "Painting"],
+    phone: '', email: '',
+    aboutUs: 'blah blah blah',
+  ),
+  const ProfileCard(
+    id: 5,
+    companyName: "Cool Air HVAC",
+    ownerName: "Sarah Brown",
+    image: "/placeholder.svg?height=100&width=100",
+    rating: 4.7,
+    tags: ["HVAC", "Air Conditioning", "Heating"],
+    phone: '', email: '',
+    aboutUs: 'blah blah blah',
+  ),
+  const ProfileCard(
+    id: 5,
+    companyName: "Cool Air HVAC",
+    ownerName: "Sarah Brown",
+    image: "/placeholder.svg?height=100&width=100",
+    rating: 4.7,
+    tags: ["HVAC", "Air Conditioning", "Heating"],
+    phone: '', email: '',
+    aboutUs: 'blah blah blah',
+  ),
+];
+
+
+class ResultsPage extends StatelessWidget{  //defined class that extends statefulwidget (means app changes over time)
+  const ResultsPage({super.key}); //constructor for creating the class (when creating a new object from the class the cunstroctor is automatically called to initialize it in which the constructor is passing information (like the key) when creating widgets)
+  //the super key is a key used by flutter to keep track of widgets in which super refers to the parent class in which it is passing the key of the constructor of the stateless widget. Key is being used to uniquely identify widgets
+  @override
+  Widget build(BuildContext context) { //build function is used to describe how the UI should look like, BuildContext gives info about the location of the widget in the app tree
+    return Scaffold(
+      appBar: AppBar(
+      ),
+      body: Column(
+        children: [
+          Padding(
+            padding: const EdgeInsets.all(9.0),
+            child: SearchBar(
+            hintText: 'Search',
+              onChanged: (value) {},
+            ),
+          ),
+          Expanded(
+          child: ListView.builder(
+          padding: const EdgeInsets.symmetric(horizontal: 20.0),
+            itemCount: contractor.length,
+            itemBuilder: (context,index){
+              return contractor[index];
+            },
+          ),
+         ),
+         const Row(children: [
+          Text("End of Results")
+         ],)
+      ],
+    ),
+  );
+}
+}
+
+//creating a class for the tags
+//this can be added later: this widget accepts text and color, in which depending on the tag, the color of the oval should be different
+class OvalTags extends StatelessWidget {
```

Comment:
did we not move this to another page already? Might just need to merge the other branch and these changes will update